tests/e2e/specs/blocks/shopper/shopper-wc-blocks-checkout-failures.spec.js

```diff
@@ -44,6 +44,7 @@ describeif( RUN_WC_BLOCKS_TESTS )(
 
 		afterEach( async () => {
 			// Clear card details for the next test
+			await shopperWCP.openCheckoutWCB();
```

Comment:
Explanation for this change is here p1650537588903029/1649998135.891069-slack-CQ0Q6N62D